src/event-handler/on.ts

```diff
@@ -29,18 +29,20 @@ export function receiverOn(
     return;
   }
 
-  if (emitterEventNames.indexOf(webhookNameOrNames) === -1) {
-    console.warn(
-      `"${webhookNameOrNames}" is not a known webhook name (https://developer.github.com/v3/activity/events/types/)`
-    );
+  if (["*", "error"].includes(webhookNameOrNames)) {
+    const webhookName =
+      (webhookNameOrNames as string) === "*" ? "any" : webhookNameOrNames;
+
+    const message = `Using the "${webhookNameOrNames}" event with the regular Webhooks.on() function is not supported. Please use the Webhooks.on${
+      webhookName.charAt(0).toUpperCase() + webhookName.slice(1)
+    }() method instead`;
+
+    throw new Error(message);
```

Comment:
Let's get rid of this altogether. It isn't needed anymore

Reply:
This was added [at the request](https://github.com/octokit/webhooks.js/pull/444#issuecomment-774256384) of @gr2m :)